.github/workflows/build-win.yml

```diff
@@ -24,6 +24,22 @@ jobs:
         docker system prune -a -f
         docker volume prune -a -f
       shell: pwsh
+    - name: Set TAG variable based on the branch
+      run: |
+        if ($env:GITHUB_REF -match ".*release/(.*)") {
```

Comment:
do you want to check "refs/heads/release/" instead of ".*release/"?

Reply:
updated